File: laboratorna 3/src/Goods.java
public class Goods {

    private String name;
    private double price;
    private int expirationDate;
    private CategoryType categoryType;

    public Goods (String name, double price,  int expirationDate, CategoryType categoryType) {
        this.name = name;
        this.price = price;
        this.expirationDate = expirationDate;
        this.categoryType = categoryType;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public int getExpirationDate() {
        return expirationDate;
    }

    public void setExpirationDate(int expirationDate) {
        this.expirationDate = expirationDate;
    }

    public CategoryType getCategoryType() {
        return categoryType;
    }

    public void setCategoryType(CategoryType categoryType) {
        this.categoryType = categoryType;
    }

    @Override
    public String toString() {
        return "Goods{" +
                "name='" + name + '\'' +
                ", price=" + price +
                ", expirationDate=" + expirationDate +
                ", categoryType=" + categoryType +
                '}';
    }
}

File: lab-11 python/Goods.py
class Goods:

    def __init__(self, name, price, expiration_date, category_type):
        self.name = name
        self.price = price
        self.expiration_date = expiration_date
        self.category_type = category_type
    def __str__(self):
        return ' name = ' + self.name + ' price = ' \
               + str(self.price) + ' expiration_date = ' + str(self.expiration_date) +\
               ' category_type = ' + str(self.category_type)


